main.go

```diff
@@ -233,6 +251,33 @@ func fetchPages(team, since, until string) ([]*page, error) {
 	return pages, nil
 }
 
+func fetchTeamForPagerdutyIncident(client *pagerduty.Client, incidentId string) (string, error) {
+	alertsResp, err := client.ListIncidentAlerts(incidentId)
+	if err != nil {
+		return "", err
+	}
+
+	for _, a := range alertsResp.Alerts {
+		if details, ok := a.Body["details"]; ok {
+			if detailsMap, ok := details.(map[string]interface{}); ok {
+				if tags, ok := detailsMap["tags"]; ok {
+					if tagsString, ok := tags.(string); ok {
+						tokens := strings.Split(tagsString, ",")
+						for _, token := range tokens {
+							cleaned := strings.TrimSpace(token)
+							if strings.HasPrefix(cleaned, "team:") {
+								return cleaned[strings.Index(cleaned, ":")+1:], nil
+							}
+						}
```

Comment:
This is the spot that could use some generalization - by filter. Since this convention can be different for different teams, and it could be interesting to filter by certain other criteria which would make it pretty powerful.